lib/bolt/config.rb

```diff
@@ -344,6 +344,12 @@ def deep_clone
     end
 
     private def update_logs(logs)
+      begin
+        FileUtils.touch(File.expand_path('bolt-debug.log', @project.path))
```

Comment:
Oh hmm, we do this before we handle the fact that the log might be disabled (and before we validate its config), so we will still create it even if the user disabled it.

It's a bit of a pain though because the loop below also calls `normalize_log`, which obscures the location of `bolt-debug.log` within the `log` map. It honestly might be easiest to just repeat the check here. We could also use that as an opportunity to verify that `bolt-debug.log` is indeed a defined log (there's no way it wouldn't be _now_, but that could change).

```rb
if logs['bolt-debug.log'] && logs['bolt-debug.log'] != 'disable'
  FileUtils.touch(File.expand_path('bolt-debug.log', @project.path))
end
```

Reply:
Also this block could probably use a comment since its intent is non-obvious.

Reply:
Oh dang, good point. I'll just do that.